roles/openshift_health_checker/test/etcd_volume_test.py

```diff
@@ -15,7 +16,7 @@ def test_cannot_determine_available_disk(ansible_mounts, extra_words):
     with pytest.raises(OpenShiftCheckException) as excinfo:
         EtcdVolume(fake_execute_module, task_vars).run()
 
-    for word in 'Unable to find etcd storage mount point'.split() + extra_words:
+    for word in ['Unable to determine mount point'] + extra_words:
```

Comment:
Same as for the other test file, note the difference word-by-word x whole sentence. If that's understood, no problem with the change.